news/tests/test_views.py

```diff
@@ -565,6 +601,83 @@ def test_success_sync_optin_lowercase(self):
             self.update_user_task.assert_called_with(request, SUBSCRIBE, data=request_data,
                                                      optin=True, sync=True)
 
+    @patch('news.views.IP_RATE_LIMIT_EXTERNAL', '1/m')
+    def test_ip_rate_limit(self):
+        self.update_user_task.return_value = HttpResponseJSON({'status': 'ok'})
+        request_data = {'newsletters': 'news,lets', 'optin': 'N', 'sync': 'N',
+                        'email': 'dude@example.com'}
+        self.client.post('/news/subscribe/', request_data,
+                         HTTP_X_CLUSTER_CLIENT_IP='1.1.1.1')
+        self.assertTrue(self.update_user_task.called)
+        self.update_user_task.reset_mock()
+
+        resp = self.client.post('/news/subscribe/', request_data,
```

Comment:
Don't follow you. This is a test specifically for the 2nd part. I don't have to test the response in the first part so well, but I do need to make sure the first request works since the rate limit should only kick in on the 2nd call.